docker-compose.yml

```diff
@@ -16,8 +15,7 @@ services:
       - DATA_DIR=${DATA_DIR-}
       - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR-}
       - LOCALSTACK_API_KEY=${LOCALSTACK_API_KEY-}  # only required for Pro
-      - HOST_TMP_FOLDER=${TMPDIR:-/tmp/}localstack
       - DOCKER_HOST=unix:///var/run/docker.sock
     volumes:
-      - "${TMPDIR:-/tmp}/localstack:/tmp/localstack"
+      - "${TMPDIR:-/tmp}/localstack:/var/lib/localstack"
```

Comment:
i would also like to rename this variable to `LOCALSTACK_VOLUME_DIR`, which matches the one we're now using in the CLI.

in fact i would like to change this line all together, to have the volume in a persistent directory, ideally in the same directory as the docker-compose, as is typical for other containers.

options:
* `     - "${LOCALSTACK_VOLUME_DIR:-./volume}:/var/lib/localstack"`
* `     - "${LOCALSTACK_VOLUME_DIR:~/.cache/localstack:/var/lib/localstack"`

WDYT?